gulpfile.js

```diff
@@ -232,7 +235,7 @@ gulp.task("serve", ["build"], function () {
     });
 });
 
-gulp.task("build", ["configs", "data", "styles", "plugins"], function (done) {
+gulp.task("build", [ "configs", "data", "styles", "plugins"], function (done) {
     jekyllBuild(done);
 });
 
```

Comment:
@nikhilkh Just a small note: these changes made it in accidentally. I'm doing a slight refactor and I'll revert them.